src/index.ts

```diff
@@ -1062,6 +1063,23 @@ export abstract class PinejsClientCore<PinejsClient> {
 		const onRetryHandler =
 			retryParameters.onRetry ?? retryDefaultParameters.onRetry;
 
+		const getRetryAfterHeaderDelayMs = (err: unknown) => {
+			const getRetryAfterHeader =
+				retryParameters.getRetryAfterHeader ??
+				retryDefaultParameters.getRetryAfterHeader;
```

Comment:
I would probably move keep this part in the closure but then move `getRetryAfterHeaderDelayMs` to the module level as something like `getRetryAfterHeaderDelayMs(getRetryAfterHeader, err)` to avoid creating it each time